backend/src/modules/match-module/core/application/command/EndMatch.ts

```diff
@@ -0,0 +1,9 @@
+export class EndMatch {
+  readonly matchId: string;
+  readonly winner: string;
```

Comment:
Chyba bym zostawił nazwę winnerId w command. Co innego jeśli mamy typ: `winner: MatchSideId`, no to wtedy wiemy w jaki sposób jest winner opisany. Teraz trochę mi się tak kłóci matchId z id i winner bez id.

Reply:
to może wszędzie zmienić na winnerId i looserId, żeby nie było żadnych wątpliwości ?